Files/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -319,33 +307,30 @@
                                 <ColumnDefinition x:Name="Column8" Width="0" />
                             </Grid.ColumnDefinitions>
 
-                            <Grid Grid.Column="0" Width="44" />
```

Comment:
Can you revert this change?

Reply:
Nope if I do the columns headers and contents are not aligned

Reply:
I'll take a look again. It seems like some folders have a different width for that column.

Reply:
Maybe because it has been saved in preferences for some reason? You coud try disabling "Enable individual preferences for individual folders"

Reply:
That seems to cause other issues.
![image](https://user-images.githubusercontent.com/39923744/120114781-bac4c680-c14e-11eb-9a06-67a1b5541738.png)

Reply:
> Maybe because it has been saved in preferences for those folders? You could try disabling "Enable individual preferences for individual folders" and see if they are all the same width.

Good catch, I'll give that a try.

Reply:
Can we prevent the user setting from being changed for the icon column?